scripts/sensor/03-time_frequency.py

```diff
@@ -77,15 +79,37 @@ def run_time_frequency(subject, session=None):
         itc.save(itc_fname_out, overwrite=True)
 
 
+def get_config(
+    subject: Optional[str] = None,
+    session: Optional[str] = None
+) -> BunchConst:
+    cfg = BunchConst(
+        task=config.get_task(),
+        datatype=config.get_datatype(),
+        acq=config.acq,
+        rec=config.rec,
+        space=config.space,
+        deriv_root=config.get_deriv_root(),
+        time_frequency_conditions=config.time_frequency_conditions,
+        analyze_channels=config.analyze_channels,
+        spatial_filter=config.spatial_filter,
+        ch_types=config.ch_types,
+        eeg_reference=config.eeg_reference,
```

Comment:
you miss time_frequency_freq_min and time_frequency_freq_max

Reply:
Fixed